lib/aws/rails/sqs_active_job/deduplication.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Aws
+  module Rails
+    module SqsActiveJob
+      extend ActiveSupport::Concern

```

Comment:
Good use of concern here. Are there any other SQS active job related examples in this gem we can clean up as such?